pinot-spi/src/main/java/org/apache/pinot/spi/ingestion/batch/IngestionJobLauncher.java

```diff
@@ -143,7 +144,7 @@ private static void kickoffIngestionJob(SegmentGenerationJobSpec spec, String in
       ingestionJobRunner = PluginManager.get().createInstance(ingestionJobRunnerClassName);
     } catch (Exception e) {
       throw new RuntimeException(
-          "Failed to create IngestionJobRunner instance for class - " + ingestionJobRunnerClassName, e);
+              "Failed to create IngestionJobRunner instance for class - " + ingestionJobRunnerClassName, e);
```

Comment:
Seems this doesn't follow the [Pinot Style](https://docs.pinot.apache.org/developers/developers-and-contributors/code-setup#intellij). Can you please apply this setting and reformat all the changes?